source/Calamari.Shared/Integration/Certificates/WindowsX509CertificateStore.cs

```diff
@@ -18,14 +17,26 @@
 
 namespace Calamari.Integration.Certificates
 {
-    public class WindowsX509CertificateStore: IWindowsX509CertificateStore
+    public class WindowsX509CertificateStore : IWindowsX509CertificateStore
     {
+        readonly ILog log;
         public static readonly ISemaphoreFactory Semaphores = new SystemSemaphoreManager();
         public static readonly string SemaphoreName = nameof(WindowsX509CertificateStore);
 
         const string IntermediateAuthorityStoreName = "CA";
         public static readonly string RootAuthorityStoreName = "Root";
 
+        public WindowsX509CertificateStore(ILog log)
+        {
+            this.log = log;
+        }
+
+        // This should only be used in tests
+        public WindowsX509CertificateStore(): this(ConsoleLog.Instance)
```

Comment:
We want to pass through the `ILog` to help with tests looking for specific output, but other logs dont care and so this default constructor makes it a little easier for those scenarios.